test/org/sagebionetworks/bridge/services/EmailVerificationServiceTest.java

```diff
@@ -27,130 +35,187 @@
 
     private static final String EMAIL_ADDRESS = "foo@foo.com";
     
+    private static final String EMAIL_ADDRESS_KEY = EMAIL_ADDRESS + ":emailVerificationStatus";
+
     @Mock
     private AmazonSimpleEmailServiceClient sesClient;
     @Mock
     private GetIdentityVerificationAttributesResult result;
     @Mock
     private IdentityVerificationAttributes attributes;
-
+    @Mock
+    private CacheProvider cacheProvider;
+    @Spy
     private EmailVerificationService service;
-    
+
     private ArgumentCaptor<GetIdentityVerificationAttributesRequest> getCaptor;
-    
+
```

Comment:
There seems to be a lot of whitespace and formatting changes in this file. Can you turn off auto-format? All the formatting changes is making it difficult to find the actual changes in this file.